_posts/2022-06-01-snitching-on-your-colleagues-using-cloud-magic.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: post
+authors: [lander_marien]
+title: 'Snitching On Your Colleagues Using Cloud Magic'
+image: /img/2022-06-01-snitching-on-your-colleagues-using-cloud-magic/Betrayal.jpg
+tags: [Spring, Spring Boot, Git, DevOps, Docker, Kubernetes, Terraform, Helm, Azure, Internet Of Things, RaspberryPi, Javascript, Node.js, AWS, DynamoDB, RDS, Postgres]
+category: Internship
+comments: true
+---
+
+
+
+# Table Of Contents
+
+* [Introduction](#introduction)
+  
+* [The Task At Hand](#the-task-at-hand)
+* [Exploring The Environment](#exploring-the-environment)
```

Comment:
I'd reduce / remove the amount of links. 1 or 2 levels deep is enough. The ToC is now too long.